cmd/strawToBedpe/strawToBedpe.go

```diff
@@ -0,0 +1,70 @@
+// Command Group: "Data Conversion"
```

Comment:
This is a random question not really pertaining to the PR directly, but have straw files already gone through sequencing filters and expected/observed interaction normalization for distance between bins? I know there can be quite significant corrections for hic sequencing data (though I don't know exactly what they entail) and it would be good to know if these have already been applied at the 'straw' stage as it will effect interaction calling I'm sure

Reply:
From my understanding, you can get any of this information from the straw file but you can also set everything to get the raw data which I believe I've done and I have an example usage listed in the description here.